src/promptflow/promptflow/_cli/_pf/_run.py

```diff
@@ -546,11 +548,15 @@ def create_run(create_func: Callable, args):
         column_mapping = list_of_dict_to_dict(column_mapping)
 
     if file:
+        # for file name items `flow` and `data`, convert them to relative path to the file
+        # data also has a special format `azureml:`, which should not be converted
         for param_key, param in {
             "name": name,
-            "flow": flow,
+            "flow": os.path.relpath(Path(flow), Path(file).parent) if flow else None,
```

Comment:
Why change here? When load run from YAML, the base path will be set to file's folder

Reply:
let me check, I suspect we could change this by paramter --flow